operations/node_management/Wipe_NCN_Disks.md

```diff
@@ -16,18 +16,20 @@ For wiping Linux on an NCN with a previously installed OS, the [basic wipe](#bas
 
 ## Topics
 
-- [Basic Wipe](#basic-wipe)
-- [Advanced Wipe](#advanced-wipe)
-- [Full Wipe](#full-wipe)
+- [Basic wipe](#basic-wipe)
+- [Advanced wipe](#advanced-wipe)
+- [Full wipe](#full-wipe)
 
-## Basic Wipe
+## Basic wipe
 
 This wipe erases the magic bits on the disk to prevent them from being recognized and making them ready for deployment, as well as removing the common volume groups.
 
 1. (`ncn#`) List the disks for verification.
 
     ```bash
-    disks_to_wipe="$(lsblk -l -o SIZE,NAME,TYPE,TRAN | grep -E '(raid|'"$metal_transports"')' | sort -u | awk '{print "/dev/"$2}' | tr '\n' ' ' | sed 's/ *$//')"
+    disks_to_wipe=$(lsblk -l -o NAME,TYPE,TRAN | grep -E '[[:space:]].*(sata|nvme|sas|raid)' |
```

Comment:
Does the grep need the `[[:space:]]`?

Also, I understand that a 1.0 user would use this 1.3 doc for wiping nodes _but_ for 1.3 we don't manually wipe anymore for reinstalls. I think 1.3 and main could still do this:

```bash
. /usr/lib/dracut/modules.d/90metalmdsquash/metal-lib.sh
.. grep -E '(raid|'"$metal_transports"')' ..
```
(where `..` is the rest of the command

Reply:
The `[[:space:]]` part is so that it doesn't match on a device name that happens to have one of those strings in it. I assumed we were only wanting to match strings found in the type or transport fields. Is that not accurate?

For 1.3/main, they may be installing a system that was previously installed with csm-1.0, for example. In that case, that library file will not exist on the system. That's why the wipe page cannot assume it.

Reply:
Yeah I just mean that if a user were to land on this page, it wouldn't be in the context of being on an older image. The upgrade path to 1.3 wouldn't use this, nor would a re-installation or fresh installation. In 1.3 and newer this page is purely operational/trouble-shooting.

For upgrades to anything else (e.g. to 1.2 or 1.0) this page is mandatory and does need to run on an older image.

Reply:
Sure. I just wasn't sure if we'd rather provide general instructions that will work on a broader range of situations, just in case it's needed, even if that isn't expected. But I'm fine with altering it for 1.3 and main and adding a note about it only working on 1.3+ NCN images, if you prefer. Would we need to update the text that says it will work from the emergency shell in that case? Or would that library be available in that context?

Reply:
We would definitely need to remove the text that says 
```text
For wiping Linux on an NCN with a previously installed OS, the [basic wipe](#basic-wipe) is sufficient.
```

Reply:
Would also need to change this line
```text
Forcing a node to net-boot that may or may not have a working or desired operating system installed (e.g. a node that might have a non-CSM Linux distro installed)
```

Reply:
For the emergency shell they need to source `/lib/metal-lib.sh` instead.

I think we should source the lib for 1.3+ just to remove the duplicated pattern *going forward*. I'll live with the previous releases, and besides we'll never patch one of our previous releases to support more transports.

Reply:
Ok, I'll make the edits. One sec.

Reply:
Done. Let me know what you think.

Reply:
Looks good. nit: it uses `source` for one and `.` for another I'd just make them consistent. Approving this tho.